charts/images.yaml

```diff
@@ -89,7 +89,7 @@ images:
 - name: konnectivity-server
   sourceRepository: github.com/gardener/replica-reloader
   repository: eu.gcr.io/gardener-project/gardener/replica-reloader
-  tag: "v0.1.0"
+  tag: "v0.2.0-konnectivity-server-v0.0.15"
```

Comment:
I do not see tag/release/branch related to this version here https://github.com/gardener/replica-reloader. Why?

Reply:
Ok, it is mentioned here https://github.com/gardener/replica-reloader#docker-images.